starfish/image/_stack.py

```diff
@@ -156,18 +156,18 @@ def _build_slice_list(
         return tuple(slice_list), axes
 
     @property
-    def raw_shape(self) -> typing.Optional[list]:
+    def raw_shape(self) -> Optional[list]:
         if self._data is None:
             return None
 
         return self._data.shape
 
     @property
-    def shape(self) -> typing.Optional[dict]:
+    def shape(self) -> Optional[dict]:
         if self._data is None:
             return None
 
-        result = collections.OrderedDict()
+        result = collections.OrderedDict()  # type: collections.OrderedDict[Any, str]
```

Comment:
Are these comments parsed without requiring python 3.6?

Reply:
Yes, this is the 3.0+ compatible style.